src/components/menues/SideMenu/SideMenu.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
next rules are applicable everywhere
1) use consisntent import order and spacing. At the very top go imports of 3rd party / external modules. Then go local modules imports. At the end usually styles are being imported.

2) use camelCase for variable names. Navigation -> navigation; WeatherInfo -> weatherInfo . etc